nano/node/bootstrap/bootstrap_bulk_pull.cpp

```diff
@@ -223,7 +223,7 @@ void nano::bulk_pull_client::received_block (boost::system::error_code const & e
 			}
 			// Is block expected?
 			bool block_expected (false);
-			bool unconfirmed_account_head (pull_blocks == 0 && pull.retry_limit != std::numeric_limits<unsigned>::max () && expected == pull.account_or_head && block->account () == pull.account_or_head);
+			bool unconfirmed_account_head (connection->node->flags.disable_legacy_bootstrap && pull_blocks == 0 && pull.retry_limit != std::numeric_limits<unsigned>::max () && expected == pull.account_or_head && block->account () == pull.account_or_head); // Used only for lazy destinations (if legacy bootstrap is not available, see nano::bootstrap_attempt::lazy_destinations_increment (...))
```

Comment:
Very minor: long line here, could put the comment above

Reply:
Updated